otter/log/spec.py

```diff
@@ -102,6 +102,7 @@ def length_calc(e):
     # event -> [(event, format_str)]
     "add-server-clb": ("Adding {server_id} with IP address {ip_address} "
                        "to CLB {clb_id}"),
+    "beg-exec-convergence": "Beginning to execute convergence",
```

Comment:
Just nitpicks: given that we don't do abbreviations so far (plus "beg" kinda sounds like begging ;) I think it'd be better to call this 'begin-convergence', which is just as short